cmd/root_test.go

```diff
@@ -153,6 +159,36 @@ func (suite *rootCmdSuite) testExecute() {
 	suite.assert.Contains(buf.String(), "blobfuse2 version")
 }
 
+func (suite *rootCmdSuite) TestParseArgs() {
+	defer suite.cleanupTest()
+	var inputs = []osArgs{
+		{input: "mount abc", output: "mount abc"},
+		{input: "mount abc --config-file=./config.yaml", output: "mount abc --config-file=./config.yaml"},
+		{input: "help", output: "help"},
+		{input: "--help", output: "--help"},
+		{input: "version", output: "version"},
+		{input: "--version", output: "--version"},
+		{input: "version --check=true", output: "version --check=true"},
+		{input: "mount abc --config-file=./config.yaml -o ro", output: "mount abc --config-file=./config.yaml -o ro"},
+		{input: "abc", output: "mount abc"},
+		{input: "-o", output: ""},
+		{input: "", output: ""},
+
+		{input: "/home/mntdir -o rw,--config-file=config.yaml,dev,suid", output: "mount /home/mntdir -o rw,dev,suid --config-file=config.yaml"},
```

Comment:
In case of mount using fstab we observed these parameters are coming as input. In case of v1 as well these are coming but there command parsing is done by libfuse and it ignores whatever is not required. In case of v2 we parse and if there are unsupported options we throw back failure, so we have added some of these options as ignorable options.

Reply:
I mean does this code handle CLI like this ? -o rw,suid,dev as well as -o rw -o suid -o dev

Reply:
yes it does. "-o rw,suid,dev" format we receive when fstab is mounting using blobfuse.

Reply:
even we can receive "-o rw,suid,--config-file=config.yaml,dev,allow_other" which we will convert to "-o rw,suid,dev,allow_other --config-file=config.yaml"